overlays/dnf5-ci/overlay.yml

```diff
@@ -10,7 +10,7 @@ components:
 
   - name: libdnf
```

Comment:
There is a couple more problems, the `name` also needs to be set to `dnf5` (for both overlays).
And `rpm-gitoverlay` trips up because there is no `git tag` in the new repo. I am looking into fixing `rpm gitoverlay`.

Reply:
I had the code fixed locally. I pushed an update